index.Rmd

```diff
@@ -183,14 +186,21 @@ import pandas as pd
 data = {'name': ['a', 'b', 'c'],
         'x': [173994.1578792833, 173974.1578792833, 173910.1578792833],
         'y': [444135.6032947102, 444186.6032947102, 444111.6032947102]}
-# Turn the data into a Pandas DataFrame, column names are extracted automatically
+
+# Turn the data into a Pandas DataFrame (column names are extracted automatically)
 df = pd.DataFrame(data)
+
+# Inspect the DataFrame
 print(df.head)
 
 # Use the coordinates to make shapely Point geometries
 geometry = [Point(xy) for xy in zip(df['x'], df['y'])]
+
 # Pandas DataFrame and shapely Points can together become a GeoPandas GeoDataFrame
-wageningenGDF = gpd.GeoDataFrame(df, geometry=geometry, crs = "EPSG:28992") # note that we here specify the CRS (projection) directly while reading to GDF
+# Note that we specify the CRS (projection) directly while reading to GDF
```

Comment:
"reading to GDF" sounds strange, as we are not really reading anything here. Perhaps "creating a GDF"?